fastlane/Fastfile

```diff
@@ -348,12 +446,37 @@ platform :ios do
                                })
   end
 
+  def setup_code_signing
+    disable_automatic_code_signing(path: './ios/Mattermost.xcodeproj')
+
+    ENV['MATCH_APP_IDENTIFIER'].split(',').each do |id|
+      target = 'Mattermost'
+      if id.include? 'NotificationService'
+        target = 'NotificationService'
+      elsif id.include? 'MattermostShare'
+        target = 'MattermostShare'
+      end
+
+      profile = "sigh_#{id}_#{ENV['MATCH_TYPE']}"
+
+      update_project_provisioning(
+          xcodeproj: './ios/Mattermost.xcodeproj',
+          profile: ENV["#{profile}_profile-path"], # optional if you use sigh
```

Comment:
I see several options, maybe there should be a lane for circle and one for devs, so one is fixed in how it should be, and the other one is flexible to match whatever configuration the dev has in his computer.

Reply:
this applies to everything, dev builds, releases, etc.. on circleCI and outside of circleCI, this actually fixes a long term complain about people having to use XCode to get around some signing issues.. but this should only be executed if MATCH is being executed, so I'll fix that.